### PR TITLE
Add changelog and update FQDN

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Update FQDN and add missing changelog https://github.com/ansible-collections/amazon.aws/pull/997

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

cloudtrail

### CODE REVIEW

File: changelogs/fragments/migrate_cloudtrail.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+major_changes:
+- rds_snapshot_info - The module has been migrated from the ``community.aws`` collection.
```
Comment (jillr) [inline changelogs/fragments/migrate_cloudtrail.yml:2]:
```suggestion
- cloudtrail - The module has been migrated from the ``community.aws`` collection.
```

**Review by jillr — COMMENTED** [review]
lgtm once the request change is made

File: changelogs/fragments/migrate_cloudtrail.yml (hunk 1 above)
Comment (alinabuzachis) [reply changelogs/fragments/migrate_cloudtrail.yml]: Done, thanks! Copy paste issue!

**Review by gravesm — APPROVED** [review]

**Comment by goneri** [comment]
thank you @alinabuzachis.

**Review by goneri — APPROVED** [review]

**Review by tremble — APPROVED** [review]

**Pull request merged**